### DISCUSSION TITLE
Process: minimum threshold for lazy consensus

### DISCUSSION THREAD

**Discussion by jacoscaz** [w3c/WebID#50] [General]
_/chair hat on_

Hi all!

PR https://github.com/w3c/WebID/pull/49 is an attempt at specifying our use of lazy consensus. Part of it focuses around introducing a "minimum participation threshold" to mitigate the risk of a PR being merged after the review period whilst having been reviewed by few or none of us.

However, I've also noticed that some of you actively follow the conversation while only intervening if there's something that they do not agree with, embracing silence as implicit assent. When I get a feeling that this might be the case I tend to explicitly solicit feedback by tagging, which (so far) has worked very well. Amongst those that I tag, I try to include at least a couple of people who I think might disagree with the change as lazy consensus is somewhat biased against dissent.

I'm happy to continue like this but I can't help but wonder if the group would prefer to specify lazy consensus in a different way? As of right now,  #49 sets the minimum participation threshold at _half of the number of participants who were active in the six months prior_ to the proposed change. If the group is happy to continue with the chair tagging people for feedback, it might make more sense to reduce that threshold to 1/3 of active participants as I fear keeping it at half might prove too high a threshold.

What do you think?

EDIT: superseded. We decided to drop the threshold altogether. Will lock this discussion in the next few days unless anyone objects.

**Comment by webr3** [comment]
Perhaps easier to measure dissent, with a finite number, and assume others, or approve?
